extensions/amp-analytics/0.1/vendors.js

```diff
@@ -458,6 +459,15 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'adobeanalytics_nativeConfig': {
```

Comment:
I've added an example in analytics.amp.html. We already have a link to our documentation in amp-analytics.md.